app/models/embedded_ansible_worker/runner.rb

```diff
@@ -36,6 +38,17 @@ def setup_ansible
     _log.info("calling EmbeddedAnsible.start")
     EmbeddedAnsible.start
     _log.info("calling EmbeddedAnsible.start finished")
+
+    5.times do
```

Comment:
hmmm, it returned to you but it wasn't ready yet?  Can we try to hit it using curl or something?